### DISCUSSION TITLE
Using Tooltip on MenuItem - Fluent UI V9

### DISCUSSION THREAD

**Discussion by maayanShtengel** [microsoft/fluentui#26056] [Q&A] [answered]
Hi 
I am trying to show Tooltip on on items of a Menu which is triggered by MenuButton.
Each MenuItem I wrapped with Tooltip component. ( all components from Fluent UI V9)
The result I get is that the Tooltip of the first item in the Menu  is displayed with opening of the Menu (meaning after clicking on the MenuButton to trigger the opening). Seems like a focus selector ( or maybe something is passed to the Tooltip of the first item and causes it to be triggered).
When moving the mouse over items in the Menu all tooltips work fine but I want to prevent this "automatic" display of the first.

Is there a way to solve it? I want the tooltip to be displayed only on hover on items in the menu. 

That is how I rendered a menuItem
![image](https://user-images.githubusercontent.com/121117370/208857106-c0deac93-f39f-4dba-98da-52920c96f2f4.png)

**Comment by layershifter** [comment, marked as answer]
There is a prop called `unstable_disableAutoFocus` for `Popover` (#26141) that prevents this behavior, it seems that it's reasonable to implement for `Menu`, too. @maayanShtengel would you mind to create an issue?

**Reply by maayanShtengel** [reply]
Thanks,  Sure.
https://github.com/microsoft/fluentui/issues/26575

**Comment by smhigley** [comment]
Just want to note that if a team does this, they would also need to set focus within the menu manually when it is opened. From an accessibility perspective, it is not at all recommended to set this to anything other than the first menu item.

In practice, without breaking keyboard and screen reader accessibility, the only way around this would be to track whether the menu is opened with a click or a keyboard event, and control the display of the tooltip based on that. The other approach would be to not display tooltips within a menu (which is what I would personally recommend in almost all cases).

**Reply by maayanShtengel** [reply]
Thanks you Sarah
